examples/fortran/simple/m_ser.f90

```diff
@@ -89,7 +93,8 @@ SUBROUTINE deserialize_with_perturb(a)
     rprecision = 10.0**(-PRECISION(1.0))
 
     ! setup serialization environment
-    call ppser_initialize(directory='.',prefix='SerialboxTest-output',prefix_ref='SerialboxTest',rprecision=rprecision,rperturb=1.0e-5_8)
+    call ppser_initialize(directory='.',prefix='SerialboxTest-output', &
+          prefix_ref='SerialboxTest',rprecision=rprecision,rperturb=1.0e-5_8)
```

Comment:
There seems to be an issue with tabs.

Reply:
@havogt  This line is still not aligned correctly

Reply:
How can a line be incorrectly aligned if there is no tool to do it? ;)

Can we try this one: https://github.com/pseewald/fprettify

Reply:
No we cannot...

Reply:
because of serialbox statements ;)

Reply:
We can use our eyes which are good for this as well. No just kidding, I just wanted to raise the point that if we change the formatting at least we try to be consistent. And do the same on all changes or just do nothing. Otherwise, I don't see the point ...